bloom/block_test.go

```diff
@@ -0,0 +1,39 @@
+package bloom_test
+
+import (
+	"math"
+	"testing"
+
+	"github.com/segmentio/parquet-go/bloom"
+)
+
+func TestBlock(t *testing.T) {
+	for i := uint64(0); i < math.MaxUint32; i = (i * 2) + 1 {
+		x := uint32(i)
+		b := bloom.Block{}
+		b.Insert(x)
+		if !b.Check(x) {
```

Comment:
Should we check x-1 or something as well

Reply:
It's tricky, because bloom filters may have false positives, `Check` may return `true` even if the key wasn't inserted.

Maybe I can revisit the test to do a bunch of checks that validate that the false positive rates are below the expected threshold.

Reply:
I added your suggestion in be1a59b